components/micro-gateway-core/src/main/ballerina/src/gateway/services/throttle_event_listener.bal

```diff
@@ -84,7 +84,7 @@ public function startSubscriberService() returns @tainted jms:MessageConsumer |
             return session;
         } else {
             jms:Destination dest = check session->createTopic("throttleData");
-            jms:MessageConsumer | error subscriberEndpoint = trap session->createDurableSubscriber(dest, "sub-1");
+            jms:MessageConsumer | error subscriberEndpoint = trap session->createConsumer(dest, "");
```

Comment:
Why we need to pass empty string. I think we can use
```java
createConsumer(dest);
```

Reply:
Fixed